deploy/mcsapi/crds/embed.go

```diff
@@ -0,0 +1,29 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+
+Copyright Contributors to the Submariner project.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package crds
+
+import _ "embed"
+
+var (
+	//go:embed multicluster.x_k8s.io_serviceexports.yaml
+	ServiceExportCRD []byte
+
+	//go:embed multicluster.x_k8s.io_serviceimports.yaml
+	ServiceImportCRD []byte
+)
```

Comment:
Perhaps we should wait for https://github.com/kubernetes-sigs/mcs-api/pull/115 and use the u/s CRDs...  Our copies match u/s.